docs/theming.md

```diff
@@ -72,68 +71,106 @@ MDCButton *button = [[MDCButton alloc] init];
 ```
 <!--</div>-->
 
-### Creating a container scheme
+### How to customize a container scheme
 
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 
 ```swift
-import MaterialComponents.MaterialColorScheme
-import MaterialComponents.MaterialTypographyScheme
-import MaterialComponents.MaterialShapeScheme
 import MaterialComponentsBeta.MaterialContainerScheme
 
 let containerScheme = MDCContainerScheme()
-containerScheme.colorScheme = myColorScheme
-containerScheme.typographyScheme = myTypographyScheme
-containerScheme.shapeScheme = myShapeScheme
+
+// You can directly configure scheme properties:
+containerScheme.colorScheme.primaryColor = .red
+
+// Or assign a customized scheme instance:
+let shapeScheme = MDCShapeScheme()
+containerScheme.shapeScheme = shapeScheme
 ```
 
 #### Objective-C
 
 ```objc
-#import <MaterialComponents/MaterialColorScheme.h>
-#import <MaterialComponents/MaterialShapeScheme.h>
-#import <MaterialComponents/MaterialTypographyScheme.h>
 #import <MaterialComponentsBeta/MaterialContainerScheme.h>
 
 MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
-containerScheme.colorScheme = self.myColorScheme;
-containerScheme.shapeScheme = self.myShapeScheme;
-containerScheme.typographyScheme = self.myTypographyScheme;
+
+// You can directly configure scheme properties
+containerScheme.colorScheme.primaryColor = [UIColor redColor];
+
+// Or assign a customized scheme instance:
+let shapeScheme = MDCShapeScheme()
+containerScheme.shapeScheme = shapeScheme
 ```
 <!--</div>-->
 
-### Using a subsystem scheme
+### Recommended theming patterns
+
+The most expedient solution to adopting container schemes and theming extensions is to create a
```

Comment:
I would try to use simpler words when possible. Instead of expedient maybe use "fastest", "simplest", "practical"?